drivers/interrupt_controller/Kconfig.tms570_vim

```diff
@@ -0,0 +1,15 @@
+# Copyright (C) 2025 ispace, inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if CPU_CORTEX_R5
+
+config TMS570_VIM
+	bool "TI Vectored Interrupt Manager (TMS570)"
+	default y
+	depends on DT_HAS_TI_TMS570_VIM_ENABLED
+	help
+		The TI Vectored Interrupt Manager provides hardware assistance for prioritizing
```

Comment:
help indent is 1x tab followed by 2x spaces

Reply:
fixed